lib/matplotlib/backends/backend_svg.py

```diff
@@ -1,17 +1,44 @@
+'''
+This backend renders figures to `Scalable Vector Graphics <https://www.w3.org/Graphics/SVG/>`_, an
+XML-based graphics format. This format is an `W3C` open web standard for creating vector graphics
```

Comment:
It doesn't appear that "W3C" is a link, so this should be double-backticks for code style, though maybe asterisks for bold/emphasis would make more sense. This applies to many other words in this docstring.